Source/.editorconfig

```diff
@@ -0,0 +1,25 @@
+﻿# EditorConfig is awesome: http://EditorConfig.org

```

Comment:
Well, I guess it can't hurt to include the link, in case someone sees this file and wonders what it is ;)